docs/docs/recipes.md

```diff
@@ -770,6 +770,100 @@ export const pageQuery = graphql`
 - [Guide to creating pages from data programmatically](/docs/programmatically-create-pages-from-data/)
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing from Wordpress
+
+#### Prerequisites
+
+- An existing [Gatsby site](/docs/quick-start/) with a `gatsby-config.js` and `gatsby-node.js` file
+- An existing template component such as `src/templates/post.js`
+
+#### Directions
+
+1. Install the `gatsby-source-wordpress` plugin by running the following command:
+
+```shell
+npm install gatsby-source-wordpress
+```
+
+2. Configure the plugin by modifying the `gatsby-config.js` file such that it looks like the following:
+
+```JS:title=gatsby-config.js
+module.exports = {
+  ...
+  plugins: [
+    ...,
```

Comment:
```suggestion
```
I think removing this is a good idea, so people don't get confused by invalid code!

Reply:
I like this notation so people don't expect that it is the only plugin to install. In some cases helper plugins are needed

Reply:
If we make it as a comment then it is valid code?

Reply:
but I think this discussion is not in the scope of this PR

Reply:
Makes sense! Thanks!